app/controllers/spree/api/retailops/settlement_controller.rb

```diff
@@ -62,14 +62,30 @@ def mark_complete
           render text: @settlement_results.to_json
         end
 
+        # This gets called by RetailOps after a returned item has been
+        # received and accepted at the warehouse. We need to look elsehwere
+        # for hook when returned item has been received but can not be restocked.
```

Comment:
This is intended to be called for all returns processed in RO regardless of whether restocking is done by the returns agent.